sql/migrations/20201120223448_world.sql

```diff
@@ -0,0 +1,431 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20201120223448');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20201120223448');
+-- Add your query below.
+
+
+-- Undercity
+-- Imbued Vial - Daniel Bartlett
+DELETE FROM `npc_vendor` WHERE `entry` = 4561 AND `item` = 18256;
+
+-- Rune Thread - Daniel Bartlett
+DELETE FROM `npc_vendor` WHERE `entry` = 4561 AND `item` = 14341;
+
+-- Shiny Bauble - Daniel Bartlett
+INSERT INTO `npc_vendor` (`entry`, `item`) VALUES
+(4561, 6529);
+
+-- Small Throwing Knife - Eleanor Rusk
+DELETE FROM `npc_vendor` WHERE `entry` = 4555 AND `item` = 2947;
+
+-- Crude Throwing Axe - Eleanor Rusk
+DELETE FROM `npc_vendor` WHERE `entry` = 4555 AND `item` = 3111;
+
+-- Black Dye - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 2325;
+
+-- Grey Dye - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 4340;
+
+-- Yellow Dye - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 4341;
+
+-- Purple Dye - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 4342;
+
+-- Heaby Silken Thread - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 8343;
+
+-- Rune Thread - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 14341;
+
+-- Crystal Vial - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 8925;
+
+-- Imbued Vial - Felicia Doan
+DELETE FROM `npc_vendor` WHERE `entry` = 4775 AND `item` = 18256;
+
+-- Missing Items - Felicia Doan
+INSERT INTO `npc_vendor` (`entry`, `item`) VALUES
+(4775, 2605),
+(4775, 6260),
+(4775, 6529),
+(4775, 2324);
+
+-- Scroll of Strength - Salazar Bloch
+UPDATE `npc_vendor` SET `maxcount` = 4 WHERE `entry` = 4581 AND `item` = 954;
+
+-- Scroll of Stamina 2 - Salazar Bloch
+UPDATE `npc_vendor` SET `maxcount` = 2 WHERE `entry` = 4581 AND `item` = 1711;
+
+-- Scroll of Intellect 2 - Salazar Bloch
+UPDATE `npc_vendor` SET `maxcount` = 2 WHERE `entry` = 4581 AND `item` = 2290;
+
+-- Scroll of Agility - Salazar Bloch
+UPDATE `npc_vendor` SET `maxcount` = 5 WHERE `entry` = 4581 AND `item` = 3012;
+
+-- Pattern: Green Whelp Bracers - Joseph Moore
+INSERT INTO `npc_vendor` (`entry`, `item`, `maxcount`, `incrtime`) VALUES
+(4589, 7451, 1, 7200);
+
+-- Strong Fishing Pole - Lizbeth Cromwell
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4574 AND `item` = 6365;
+
+-- Missing Items - Elizabeth Van Talen
+INSERT INTO `npc_vendor` (`entry`, `item`, `maxcount`, `incrtime`) VALUES
+(4587, 4389, 1, 10800),
+(4587, 4404, 3, 10800),
+(4587, 4361, 2, 10800);
+
+-- Teldrassil
+-- Missing Items - Felicia Doan

```

Comment:
Comment says Felicia Doan, but creature id 3608 is Aldia. Which npc is supposed to sell these items?